### DISCUSSION TITLE
1 page at of 4 deployded

### DISCUSSION THREAD

**Discussion by Victoria-Caron** [community/community#78697] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, 

I just uploaded my website for my school project and only one of my pages index.html is functional. My other 3 pages don't have my content. I see my code in my files but everything is blank. This is the message that appears in my console 

portfolio.html:135 
       GET https://victoria-caron.github.io/dart449-final/Javascript/scripts.js net::ERR_ABORTED 404 (Not Found

Everything works smoothly on my local desktop. 

Can someone please help me with this issue?

**Comment by DeadGolden0** [comment, marked as answer]
Hello,

In your ``portfolio.html`` file on line 135, you have ```<script src="Javascript/scripts.js"></script>```

However, your folder is named ``javascript`` without a capital J.

Try replacing it with this: ```<script src="javascript/scripts.js"></script>```

Do not hesitate to mark the post as 'Answered' if my response suits you.

I hope my message is useful to you! Have a nice day!

**Reply by DeadGolden0** [reply]
You can access your script.js with this link : https://victoria-caron.github.io/dart449-final/javascript/scripts.js
